src/app/components/views/weather-view/weather-view.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { WeatherAlertResponse } from 'src/app/models/weather-alert.model';
+import { WeatherService } from 'src/app/services/weather.service';
+
+@Component({
+  selector: 'app-weather-view',
+  templateUrl: './weather-view.component.html',
+  styleUrls: ['./weather-view.component.scss']
+})
+export class WeatherViewComponent implements OnInit {
+  public weatherAlertResponse: WeatherAlertResponse
+
+
+  constructor(private weatherService: WeatherService) { }
+
+  ngOnInit(): void {
+
+    this.weatherService.getLoading().subscribe({
```

Comment:
please place this in its own method that is called in `ngOnInit()`.  it will need jsdocs and such

Reply:
working on it. Will mark resolved when i get home and can run and test to poke around and make sure it's done right